### PR TITLE
Add new review form

### PR DESCRIPTION
**Pull request by Kage-js**
Created an "add new review" form in react per Casi's notes. Made a review api endpoint.

### CODE REVIEW

File: app/controllers/api/v1/reviews_controller.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class Api::V1::ReviewsController < ApplicationController
```
Comment (CassandraNewell) [inline app/controllers/api/v1/reviews_controller.rb:1]: Need a controller test for this controller

File: app/controllers/stops_controller.rb [hunk 2]
```diff
@@ -6,7 +6,8 @@ def index
   end
 
   def show
-    render "static_views/index"
+    @stop=Stop.find(params[:id])
```
Comment (CassandraNewell) [inline app/controllers/stops_controller.rb]: I'm surprised by this change -- it looks like you're connecting to React through a new `show.html.erb`, which is fine, but why are you defining `@stop` and `@review` here, if you're only using `show.html.erb` to load React?

**Review by CassandraNewell — CHANGES_REQUESTED** [review]

File: app/controllers/api/v1/reviews_controller.rb (hunk 1 above)
Comment (myYearOfCode) [reply app/controllers/api/v1/reviews_controller.rb:1]: We have the tests in the "user_writes_review.rspec" Should we rename that?

File: app/controllers/stops_controller.rb (hunk 2 above)
Comment (myYearOfCode) [reply app/controllers/stops_controller.rb]: You are correct. Those are not needed. They have been removed.

File: app/controllers/api/v1/reviews_controller.rb (hunk 1 above)
Comment (myYearOfCode) [reply app/controllers/api/v1/reviews_controller.rb:1]: I moved this and renamed to "spec/controllers/api/v1/reviews_controller_spec.rb.' I also added a few more response format checks. Lastly I updated all of the old "should" tests to 'expect(' tests.

**Review by CassandraNewell — APPROVED** [review]

**Pull request merged**